src/service_control_client_impl.cc

```diff
@@ -172,7 +171,7 @@ void ServiceControlClientImpl::AllocateQuotaFlushCallback(
                        // Cache error response for fail close.
                        AllocateQuotaResponse error_response;
                        QuotaError client_side_error;
-                       *client_side_error.mutable_status() = util::SaveStatusAsRpcStatus(status);
+                       *client_side_error.mutable_status() = google::service_control_client::library_conversion_helper::SaveStatusAsRpcStatus(status);
```

Comment:
nit: maybe we can add `using ::google::service_control_client::library_conversion_helper::SaveStatusAsRpcStatus` and call `SaveStatusAsRpcStatus` directly without the namespace qualifier.

Reply:
Then the using statement needs to be dropped upon import, cause internally we don't have using util::SaveStatusAsRpcStatus for the most part.

Reply:
I did a quick search for `using ::util::SaveStatusAsRpcStatus`, and it seems internally we also have quite a few of these in other projects. It doesn't seems like a problem.

Reply:
It seems strange here to use a long namespace prefix.   Can  we use different function name for easy copybara replacement?

Reply:
Yeah, I think going with orionHong's suggestion makes more sense.
"using ::google::service_control_client::SaveStatusAsRpcStatus" -> "using ::util::SaveStatusAsRpcStatus" upon import.